cdbot/cogs/cyber.py

```diff
@@ -360,6 +360,73 @@ def __init__(self, bot: Bot):
         await ctx.send(f"There are {len(game_r.members)} that have completed Cyberstart Game. Out of them, "
                        f"{len(true_r.members)} have also completed Essentials and Assess.")
 
+    @command()
+    async def elitecountdown(self, ctx: Context, event_name: str = None, event_level: str = None):
+        """
+        Gets the time until elite, and shows in an embed
+        """
+
+        # Validate and normalise inputs
+        if event_name:
+            event_name = event_name.capitalize()
+            if not (event_name in ELITE_LOCATIONS):
+                try:
+                    event_name = ELITE_ALISES[event_name]
+                except KeyError:
+                    content = f"{event_name} is not a valid elite location! " \
```

Comment:
Backslash syntax is not pretty. I do not like it. It makes me sad.

```py
content = (
    f"{event_name}..."
    f"blah"
)
```

I like this.